d2/src/main/java/com/linkedin/d2/discovery/stores/toggling/TogglingPublisher.java

```diff
@@ -41,6 +49,7 @@ public class TogglingPublisher<T>
   private final PublisherWithStatus<T> _backup;
   private final PropertyEventBus<T>       _eventBus;
   private final PropertyEventBus<T>       _nullBus = new NullEventBus<>();
+  private static final Logger LOG = LoggerFactory.getLogger(TogglingPublisher.class);
```

Comment:
Nit: `static` fields are usually declared above non-static fields. Can you move this to the top of the class, like this?
```
public class TogglingPublisher<T>
{
  private static final Logger LOG = LoggerFactory.getLogger(TogglingPublisher.class);

  private final PublisherWithStatus<T> _primary;
  private final PublisherWithStatus<T> _backup;
  private final PropertyEventBus<T>       _eventBus;
  private final PropertyEventBus<T>       _nullBus = new NullEventBus<>();
```

Reply:
Done